app/factories/GameFactory.js

```diff
@@ -51,7 +51,7 @@ exports.startGameServer = function(players, gameServerPort, path, map, LobbyServ
     }   
 }
 function fillRune(){
-    const runes = new Array();
+    const runes = new Object();
```

Comment:
@molenzwiebel @MythicManiac Yeah, a genious changed the runes Array and now it's an object :trollface: 
So you might want to discuss that in GameServer project and not here